pkg/bpf/detect.go

```diff
@@ -143,3 +143,25 @@ func HasModifyReturn() bool {
 	})
 	return modifyReturn.detected
 }
+
+func LogFeatures() string {
+	feats := ""
+
+	if HasOverrideHelper() {
+		feats += "override_return "
+	}
+	if HasBuildId() {
+		feats += "buildid "
+	}
+	if HasKprobeMulti() {
+		feats += "kprobe_multi "
+	}
+	if HasModifyReturn() {
+		feats += "fmodret "
+	}
+	// cut the final ' '
+	if len(feats) != 0 {
+		feats = feats[:len(feats)-1]
+	}
```

Comment:
How about printing something like: "override_return: true, buildid: false"? It's more verbose, but it also provides more info.

Reply:
ok, also the code will be easier ;-) will change